vsts-minbuildjava-ubuntu-vm/azuredeploy.json

```diff
@@ -193,7 +193,7 @@
         "autoUpgradeMinorVersion":true,
         "settings": {
           "fileUris": [
-            "https://raw.githubusercontent.com/azure/azure-quickstart-templates/master/vsts-minbuildjava-ubuntu-vm/scripts/java-vstsbuild-install.sh"
+            "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/vsts-minbuildjava-ubuntu-vm/scripts/java-vstsbuild-install.sh"
```

Comment:
See: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#samples-that-contain-extra-artifacts-custom-scripts-nested-templates-etc for guidance on using artifacts.